### PR TITLE
Update Spanish and Portuguese translations

### PR DESCRIPTION
**Pull request by fitojb**
This adds translations for the feature added in commit 45edb280170969ca9cda7f3eee2fcb5e8ae7560e and removes obsolete strings.

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -109,269 +109,269 @@ msgid "Minimize or show previews"
 msgstr "Minimizar o mostrar previsualizaciones"
 
 #: Settings.ui.h:10
+msgid "Focus or show previews"
+msgstr "Focalizar o mostrar previsualizaciones"
```
Comment (x80486) [inline po/es.po]: Maybe: `Enfoque o mostrar vistas previas`

File: po/es.po [hunk 2]
```diff
@@ -109,269 +109,269 @@ msgid "Minimize or show previews"
...
 
-#: Settings.ui.h:18
+#: Settings.ui.h:19
 msgid "Color"
 msgstr "Color"
 
-#: Settings.ui.h:19
+#: Settings.ui.h:20
 msgid "Border color"
 msgstr "Color del borde"
```
Comment (x80486) [inline po/es.po]: `Color de borde`

File: po/es.po [hunk 3]
```diff
@@ -109,269 +109,269 @@ msgid "Minimize or show previews"
...
 
-#: Settings.ui.h:20
+#: Settings.ui.h:21
 msgid "Border width"
 msgstr "Grosor del borde"
```
Comment (x80486) [inline po/es.po]: `Grosor de borde`

**Comment by micheleg** [comment]
Thank you. Merged.

File: po/es.po (hunk 1 above)
Comment (fitojb) [reply po/es.po]: For the sake of parallelism, I'd rather use a verb form for both :) Also, in GNOME's terminology we use "previsualización" for "preview"

**Pull request merged**